src/source/PeerConnection/Rtp.c

```diff
@@ -319,35 +335,35 @@ STATUS writeFrame(PRtcRtpTransceiver pRtcRtpTransceiver, PFrame pFrame)
     if (locked) {
         MUTEX_UNLOCK(pKvsPeerConnection->pSrtpSessionLock);
     }
-    MUTEX_LOCK(pKvsRtpTransceiver->sender.statsLock);
-    pKvsRtpTransceiver->sender.outboundStats.totalEncodedBytesTarget += pFrame->size;
-    pKvsRtpTransceiver->sender.outboundStats.framesEncoded += frames;
-    pKvsRtpTransceiver->sender.outboundStats.keyFramesEncoded += keyframes;
+    MUTEX_LOCK(pKvsRtpTransceiver->statsLock);
+    pKvsRtpTransceiver->outboundStats.totalEncodedBytesTarget += pFrame->size;
+    pKvsRtpTransceiver->outboundStats.framesEncoded += frames;
+    pKvsRtpTransceiver->outboundStats.keyFramesEncoded += keyframes;
     if (fps > 0.0) {
-        pKvsRtpTransceiver->sender.outboundStats.framesPerSecond = fps;
+        pKvsRtpTransceiver->outboundStats.framesPerSecond = fps;
     }
     pKvsRtpTransceiver->sender.lastKnownFrameCountTime = now;
-    pKvsRtpTransceiver->sender.lastKnownFrameCount = pKvsRtpTransceiver->sender.outboundStats.framesEncoded;
-    pKvsRtpTransceiver->sender.outboundStats.sent.bytesSent += bytesSent;
-    pKvsRtpTransceiver->sender.outboundStats.sent.packetsSent++;
+    pKvsRtpTransceiver->sender.lastKnownFrameCount = pKvsRtpTransceiver->outboundStats.framesEncoded;
+    pKvsRtpTransceiver->outboundStats.sent.bytesSent += bytesSent;
+    pKvsRtpTransceiver->outboundStats.sent.packetsSent++;
```

Comment:
Shouldn't this be `pKvsRtpTransceiver->outboundStats.sent.packetsSent += packetsSent`?